### PR TITLE
Add unowned cluster compatiblity to integration tests

### PR DESCRIPTION
**Pull request by aclevername**
### Description

This PR adds support for running the majority of integration tests with clusters not created by eksctl. For each suite that is applicable it required two changes:

1. Create an unowned cluster in the beforesuite if `params.UnownedCluster` is true
2. Use a config file to create nodegroups as we need to provide the VPC for it to work

Triggering the `integration tests` action now runs two jobs in parallel, one for unowned cluster and one for eksctl created clusters.

I've also set the `inferentia` test suite and the GPU node test inside the `crud` suite to pending as this is flaking regardless of how the cluster was created
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: integration/tests/unowned_cluster/unowned_cluster_test.go [hunk 1]
```diff
@@ -3,24 +3,20 @@
 package unowned_clusters
```
Comment (Callisto13) [inline integration/tests/unowned_cluster/unowned_cluster_test.go:3]: does this file need to exist anymore?

Comment (aclevername) [reply integration/tests/unowned_cluster/unowned_cluster_test.go:3]: great question! I did delete this at one point, but then I realised this currently tests our support for unowned nodegroups in an unowned cluster. I think in a follow up PR I will move that test into a different suite and delete this :smile:

**Review by Callisto13 — APPROVED** [review]

**Comment by aclevername** [comment]
Not merging this until the existing integration tests on main are stable

**Comment by aclevername** [comment]
Green run :tada: https://github.com/weaveworks/eksctl/runs/2457964837?check_suite_focus=true

**Review by Callisto13 — APPROVED** [review]

**Pull request merged**